openmc/mgxs/library.py

```diff
@@ -773,9 +773,9 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
             nuclide this will be set to 'macro' regardless.
         xs_ids : str
             Cross section set identifier. Defaults to '1m'.
-        order : Scattering order for this dataset entry.  Default is None,
-            which will force the XSdata object to use whatever the maximum
-            order available.
+        order : Scattering order for this data entry.  Default is None,
+            which will force the XSdata object to use whatever the order of the
```

Comment:
what do you think about renaming the `order` property as `legendre_order` to mirror naming convention in `openmc.mgxs` as closely as possible? For the record, I initially used `order` and then changed to `legendre_order` in order to 1) be as transparent as possible and 2) avoid confusion since the word "order" can mean different things to different folks when dealing with array-based data structures (*i.e.*, similar meaning to the `row_column` parameter in `ScatterMatrixXS.get_xs(...)`). In any case, IMHO we should make the input generation for your MG mode mirror `openmc.mgxs` and vice versa as much as possible to provide a truly streamlined package.